### ISSUE TITLE
[Feature request] keep full screen from minimizing "window borderless mode"

### ISSUE THREAD

**Issue by Omniains** [exelix11/SysDVR#161]
so using a dual monitor setup, any time I click over to the other monitor or application on it, the full screen of the built-in player will minimize. 

if there is a way to keep it from minimizing after pressing F11 when clicking to the other monitor that would be really cool. I have tried a few always on top programs but they dont seem to work. while using an alternative player seems to have way more latency than id like even when following the advanced instructions for setting them up. 

basically like when I was using VLC as a way to stream it, I could go full screen with its built in option and click over to firefox and sysdvr still stay up on main monitor while firefox was still showing on second.

**Comment by exelix11** [comment]
Mh this seems to be yet another SDL bug, will try to find a way to workaround it at least on windows.

**Comment by Omniains** [comment]
much appreciated!
